js/src/lib/hooks/useFeedbackCollectionToast.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  useToast,
+  Alert,
+  AlertDescription,
+  Link,
+  Image,
+  Flex,
+  IconButton,
+} from "@chakra-ui/react";
+
+function ReactionFeedback({
+  description,
+  onLike,
+  onDislike,
+  externalLink,
+  externalLinkText,
+}: {
+  description: string;
+  onLike: () => void;
+  onDislike: () => void;
+  externalLink?: string;
+  externalLinkText?: string;
+}) {
+  return (
+    <Flex
+      gap={4}
+      justifyContent="center"
+      alignContent={"center"}
+      alignItems={"center"}
+    >
+      {description}
+      <IconButton
+        aria-label="thumbs up"
+        variant={"ghost"}
+        icon={<Image src="/imgs/feedback/thumbs-up.png" alt="like" />}
+        width={"32px"}
+        height={"32px"}
+        onClick={onLike}
+      />
+      <IconButton
+        aria-label="thumbs down"
+        variant={"ghost"}
+        icon={<Image src="/imgs/feedback/thumbs-down.png" alt="dislike" />}
+        width={"32px"}
+        height={"32px"}
+        onClick={onDislike}
+      />
+      {externalLink && externalLinkText && (
+        <Link href={externalLink} isExternal textDecoration="underline">
+          {externalLinkText}
+        </Link>
+      )}
+    </Flex>
+  );
+}
+
+export function useFeedbackCollectionToast(options: {
+  feedbackId: string;
+  description: string;
+  onFeedbackSubmit: (feedback: string) => void;
+  externalLink?: string;
+  externalLinkText?: string;
+}) {
+  const toast = useToast();
+  const {
+    feedbackId,
+    description,
+    onFeedbackSubmit,
+    externalLink,
+    externalLinkText,
+  } = options;
+
+  function feedBackCollectionToast(skipBypassFeedback: boolean = false) {
+    const isSkipFeedback = localStorage.getItem(feedbackId);
+    if (toast.isActive(feedbackId)) {
+      // Don't show the toast again if it's already active
+      return;
+    }
+    if (isSkipFeedback === "true" && skipBypassFeedback === false) {
+      return;
+    }
+
+    toast({
+      id: feedbackId,
+      position: "bottom-right",
+      duration: null,
+      isClosable: true,
```

Comment:
the `isClosable` doesn't work, not sure if it is related to the render function

Reply:
Remove this flag, due to we use the custom render. This flag won't be work when the toast is custom-rendered.